src/examples/index.ts

```diff
@@ -0,0 +1,85 @@
+import { useGather } from '../index';
+
+// can be ran directly with `npx ts-node src/examples/index.ts`
+
+const API_KEY = 'API_KEY_HERE';
+const SPACE_ID_1 = 'gL2sTLbVCqghDwB8/gathertown_js';
+const SPACE_ID_2 = 'tiWyyxlRs6Bl1dVS/test-space-1';
```

Comment:
should we really add these creds to the source files?

Reply:
this is entirely for manually testing, will remove it as well

Reply:
> this is entirely for manually testing, will remove it as well

kindly remove this, we can note this on documentation as well. Also, we can add this to the contributing documentation.